easybuild/easyconfigs/e/ensmallen/ensmallen-2.21.1-foss-2023a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'ensmallen'
+version = '2.21.1'
+
+homepage = 'https://ensmallen.org/'
+description = """ensmallen is a high-quality C++ library for non-linear numerical optimization"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
this software looks better suited for the `gfbf` toolchain, as it does not use MPI.